src/components/30-organisms/modal/index.js

```diff
@@ -0,0 +1,94 @@
+import { LitElement, html, css, unsafeCSS, svg } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import closeIcon from '@axa-ch/materials/internal-icons/close.svg';
+
+import { defineVersioned } from '../../../utils/component-versioning';
+import { applyDefaults } from '../../../utils/with-react';
+import styles from './index.scss';
+
+class AXAModal extends LitElement {
+  static get tagName() {
+    return 'axa-modal';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      open: { type: Boolean },
```

Comment:
Did you think about to "reflect" this prop? https://lit-element.polymer-project.org/guide/properties#reflected-attributes